source/playgrounds.txt

```diff
@@ -150,19 +150,41 @@ If you have a playground file open in |vscode-short| and do not have
 an active connection, |vsce| displays
 :guilabel:`Click here to add connection` at the top of your playground.
 
-.. figure:: /images/vsce-playground-add-connection-btn.png
-   :figwidth: 700px
-   :alt: Link to add connection from playground
-
-1. Click this link to open the connection string dialog.
-
-#. Enter the
-   :manual:`connection string </reference/connection-string/>` for the
-   deployment you want to run this playground against.
+.. procedure::
+   :style: normal
    
-#. Press :guilabel:`Enter`.
-
-#. Run your playground.
+   .. step:: Select :guilabel:`Click here to add connection` 
+
+      After you click :guilabel:`Click here to add connection`, |vsce| opens the 
+      connection string dialog.
+
+      .. figure:: /images/vsce-playground-add-connection-btn.png
+         :figwidth: 700px
+         :alt: Link to add connection from playground
+         
+   .. step:: Enter your connection string
+      
+      Enter the :manual:`connection string </reference/connection-string/>` for 
+      the deployment you want to run this playground against.
+
+      .. note:: 
+         
+         If the connection string specifies a database, the playground runs 
+         against the default connected database unless you explicitly call 
+         ``use('<database_name>')``  to use another database.
+
+         If you are connected to a default database, the playground 
+         autocompletes only for collection names available on the default 
+         database. 
+         
+   .. step:: Press :guilabel:`Enter`
+
+      After you enter your connection string, a confirmation message replaces 
+      the :guilabel:`Click here to add connection` link. The confirmation 
+      message includes information on the connected deployment and, if 
+      applicable, the default database. 
```

Comment:
I think you can shorten this to:

The confirmation message includes information on the connected deployment.

I dont see anything siginicant on the default database information, let me know if I am looking at the wrong spot!:

 
<img width="482" alt="Screenshot 2023-12-08 at 5 40 08 PM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/85948430/4fa5a5d3-a3a2-45d8-a657-784f39679d1c">

Reply:
The original ticket filer wanted us to make a note about this because multiple users have shared feedback that weren't aware they were in the incorrect database, which prevented them from connecting to collections in other dbs. 

<img width="792" alt="connection-with-default-connected-db" src="https://github.com/mongodb/docs-mongodb-vscode/assets/112565127/59290cd7-ab7d-488a-bbe1-3e11b6f3f704">

Reply:
I can change the wording to say "current" database rather than default.